Source/Orts.Simulation/Simulation/RollingStocks/MSTSLocomotive.cs

```diff
@@ -1853,7 +1921,16 @@ public override void Update(float elapsedClockSeconds)
             if (!AdvancedAdhesionModel)  // Advanced adhesion model turned off.
                AbsWheelSpeedMpS = AbsSpeedMpS;
 
-            UpdateTractiveForce(elapsedClockSeconds, t, AbsSpeedMpS, AbsWheelSpeedMpS);
+            // Cruise Control
+            CruiseControl?.Update(elapsedClockSeconds);
+            if (CruiseControl != null && CruiseControl.OverrideForceCalculation) CruiseControl.UpdateMotiveForce(elapsedClockSeconds, AbsWheelSpeedMpS);
+            else UpdateTractiveForce(elapsedClockSeconds, t, AbsSpeedMpS, AbsWheelSpeedMpS);
```

Comment:
This also breaks traction cut-off from TCS scripts, so definitely I'll have to refactor it. Probably this will imply eliminating the UseThrottle configuration option, so that it is the throttle what is always changed, but without moving the throttle controller.

Reply:
Eliminating the UseThrottle option would require changes in existing .eng files? It's an option I am using.

Reply:
What is the difference when that option is selected? My changes implement an intermediate solution where only the ThrottlePercent is being changed (the physical throttle controller not being moved). The option would become useless, but I don't think it would break anything.

Reply:
I was wrong. I'm not using the UseThrottle option.